packages/react/src/components/Toggletip/__tests__/Toggletip-test.js

```diff
@@ -4,23 +4,177 @@
+      const CustomComponent = forwardRef((props, ref) => {
+        return <div data-testid="custom-component" ref={ref} {...props} />;
+      });
+
+      render(<Toggletip as={CustomComponent}>test</Toggletip>);
+
+      expect(screen.getByTestId('custom-component')).toBeInTheDocument();
+    });
+
+    it('should support a custom class name with the `className` prop', () => {
+      const { container } = render(
+        <Toggletip data-testid="toggletip" className="custom-class">
+          <span>test</span>
+        </Toggletip>
+      );
+
+      expect(container.firstChild).toHaveClass('custom-class');
+    });
+    it.each([
+      'left',
+      'right',
+      'top',
+      'bottom',
+      'top-left',
+      'top-right',
+      'bottom-left',
+      'bottom-right',
+      'left-top',
+      'left-top',
+      'right-top',
+      'right-bottom',
+    ])('should support different alignments with the `align` prop', (align) => {
+      const { container } = render(
+        <Toggletip data-testid="toggletip" align={align}>
+          <span>test</span>
+        </Toggletip>
+      );
+
+      expect(container.firstChild).toHaveClass(`${prefix}--popover--${align}`);
+    });
+    it('should initially be open if `defaultOpen` is set to true', () => {
+      const { container } = render(
+        <Toggletip data-testid="toggletip" defaultOpen>
+          <span>test</span>
+        </Toggletip>
+      );
+
+      expect(container.firstChild).toHaveClass(`${prefix}--toggletip--open`);
+      expect(container.firstChild).toHaveClass(`${prefix}--popover--open`);
+    });
+
+    it('should close when the browser window loses focus', () => {
+      const { container } = render(
+        <Toggletip data-testid="toggletip" defaultOpen>
+          <span>test</span>
+        </Toggletip>
+      );
+
+      fireEvent.blur(window);
+
+      expect(container.firstChild).not.toHaveClass(
+        `${prefix}--toggletip--open`
+      );
+      expect(container.firstChild).not.toHaveClass(`${prefix}--popover--open`);
+    });
+
+    it.todo(
+      'should return to the trigger button if the menu is closed while focus is still inside the menu'
+    );
+
+    it('should not close when the menu itself is clicked', async () => {
+      const { container } = render(
+        <Toggletip data-testid="toggletip" defaultOpen>
+          <div data-testid="innerDiv">test</div>
+        </Toggletip>
+      );
+
+      await userEvent.click(screen.queryByTestId('innerDiv'));
+
+      expect(container.firstChild).toHaveClass(`${prefix}--toggletip--open`);
+      expect(container.firstChild).toHaveClass(`${prefix}--popover--open`);
+    });
+
+    it('should be closed when the focus leaves the tooltip', async () => {
+      const { container } = render(
+        <Toggletip data-testid="toggletip" defaultOpen>
+          <span>test</span>
+        </Toggletip>
+      );
+
+      // await userEvent.click(document.body);
+      fireEvent.blur(window);
```

Comment:
let's remove this commented out code
```suggestion
      fireEvent.blur(window);
```